<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Function Plotter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCustomPlot" name="customPlot" native="true"/>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="toolsGridLayout">
      <item row="0" column="0" colspan="3">
       <layout class="QHBoxLayout" name="functionLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="enterFunctionLabel">
          <property name="text">
           <string>Function:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="equationBox">
          <property name="font">
           <font>
            <family>Ubuntu Condensed</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="minValueLayout">
        <item>
         <widget class="QLabel" name="minValueLabel">
          <property name="text">
           <string>Min. Value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="minValueBox"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="maxValueLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="maxValueLabel">
          <property name="text">
           <string>Max. Value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maxValueBox"/>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="noPointsLayout">
        <item>
         <widget class="QLabel" name="noPointsLabel">
          <property name="text">
           <string>Number of Points:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="noPointsSpinBox">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="autoNoPointsCheckBox">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="3" rowspan="2">
       <layout class="QVBoxLayout" name="ButtonsLayout">
        <item>
         <widget class="QPushButton" name="plotButton">
          <property name="text">
           <string>Plot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="aboutButton">
          <property name="text">
           <string>About</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>noPointsSpinBox</tabstop>
  <tabstop>autoNoPointsCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
